bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java

```diff
@@ -1254,10 +1254,11 @@ public ReflectionSupport(T instance) {
 		protected Object getFieldValue(Field field) {
 			Object value = null;
 			if (field != null) {
-				boolean accessible = field.isAccessible();
+				boolean accessible = field.canAccess(instance);
 				try {
-					field.setAccessible(true);
-					value = field.get(instance);
+					if (field.trySetAccessible()) {
+						value = field.get(instance);
+					}
 				} catch (Exception exc) {
 					// do nothing
 				} finally {
```

Comment:
finally block is not symmetric anymore. I think the finally can be moved inside the `if(try)` block.

Reply:
i don't get what you mean with "symmetric" here or how moving the try block should help, but OK, won't hurt either.